Modularize, update, and enhance user focus of the "Architecture" part of Planning_for_Project (#2607)
theforeman/foreman-documentation: guides/common/modules/con_content-delivery-network-structure.adoc

[id="Content_Delivery_Network_Structure_{context}"]

> = Content Delivery Network Structure

Author: Applied.

ifdef::katello,orcharhino[]

> NOTE: The information in this section applies if you use a Red{nbsp}Hat Manifest to consume Red{nbsp}Hat content.

Reviewer: ```suggestion
NOTE: Information in this section applies only if you consume Red[nbsp]Hat content using a Red{nbsp}Hat Manifest.
```
Does this sound better to you?
